parlai/core/torch_generator_agent.py

```diff
@@ -516,6 +516,16 @@ def __init__(self, opt: Opt, shared=None):
         else:
             # this is not a shared instance of this class, so do full init
             self.criterion = self.build_criterion()
+
+            def load_init_model() -> Dict[str, Any]:
```

Comment:
how is this used?

Reply:
ahh it's not, that's an artifact. will delete